src/Raven.Server/Documents/PeriodicBackup/Aws/RavenAwsS3Client.cs

```diff
@@ -385,6 +410,18 @@ public static void FillMetadata(MetadataCollection collection, IDictionary<strin
                 collection[Uri.EscapeDataString(kvp.Key)] = Uri.EscapeDataString(kvp.Value);
         }
 
+        private static IDictionary<string, string> ConvertHeaders(HeadersCollection collection)
```

Comment:
Please note that in general there can be multiple headers with the same name. Not sure if it changes anything in this case.